model/import.go

```diff
@@ -16,12 +16,14 @@ const (
 // Import represents a completed Translation that is being imported
 // into an Installation in order to track that process
 type Import struct {
-	ID            string
-	CreateAt      int64
-	CompleteAt    int64
-	StartAt       int64
-	LockedBy      string
-	TranslationID string
+	ID             string
+	CreateAt       int64
+	CompleteAt     int64
+	StartAt        int64
+	LockedBy       string
+	TranslationID  string
+	InstallationID string
```

Comment:
Did you mean to add `InstallationID` to both `Import` as well as `ImportStatus`?

Reply:
Ah, no, this is left over from an approach to the implementation of this PR that I briefly started on and then abandoned, and missed this during cleanup, thanks.